zwave_js_server/exceptions.py

```diff
@@ -162,11 +162,22 @@ def __init__(self, value: "Value", path: str) -> None:
 class NotificationHasUnsupportedCommandClass(BaseZwaveJSServerError):
     """Exception raised when notification is received for an unsupported CC."""
 
-    def __init__(self, event: "Event", command_class: CommandClass) -> None:
+    def __init__(self, event: "Event", command_class: "CommandClass") -> None:
         """Initialize an invalid Command Class error."""
         self.event_data = event.data
         self.command_class = command_class
         super().__init__(
             "Notification received with unsupported command class "
             f"{command_class.name}: {event.data}"
         )
+
+
+class RssiErrorReceived(BaseZwaveJSServerError):
+    """Exception raised when an RSSI error is received."""
+
+    def __init__(self, error: Union["RssiError", List[int]]) -> None:
```

Comment:
Let's use separate arguments and attributes for the different types to avoid ambiguity.

Reply:
Already stale.

Reply:
I've switched it so that the conversion is handled in the exception itself